selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -40,41 +46,65 @@ def is_person_removed_from_list(self, lookup_text):
         user id
         """
         try:
-            WebDriverWait(self._driver, 30).until_not(lambda s: s.find_element(
+            WebDriverWait(self._driver, 60).until_not(lambda s: s.find_element(
```

Comment:
this extension of time is to give test a little more time to locate element which sometimes fails (why is why there is a webdriver wait  in there in the first place).  The stale element exception is in the removal of the progress check bar lines 62-63 here:  https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/pull/104/files#r60489759